pytorch_lightning/callbacks/progress.py

```diff
@@ -377,6 +389,19 @@ def on_test_end(self, trainer, pl_module):
         super().on_test_end(trainer, pl_module)
         self.test_progress_bar.close()
 
+    def on_predict_start(self, trainer, pl_module):
+        super().on_train_start(trainer, pl_module)
+        self.predict_progress_bar = self.init_train_tqdm()
+
+    def on_predict_batch_end(self, trainer, pl_module, outputs, batch, batch_idx, dataloader_idx):
+        super().on_test_batch_end(trainer, pl_module, outputs, batch, batch_idx, dataloader_idx)
+        if self._should_update(self.test_batch_idx, self.total_test_batches):
```

Comment:
Is this correct? using `on_teest_batch_end` and `total_test_batches`?

Reply:
No, it isn't :) On it !